src/zoktai/any.md

```diff
@@ -87,13 +99,15 @@ Backtrack and take the stairs, kill all Ghouls as you proceed to reach **Lvl 4**
 In the 5th room, knock on the wall near the center of the room, then hug the right wall to avoid the Zombie.  
 Use ![][icon_flame] Flame to light the torches.
 
-### <span class="puzzle">Lamp Puzzle</span>
+### <span class="puzzle">Torch Puzzles</span>
```

Comment:
Puzzle header described as "torch", main body text still reads "lamp", recommend standardizing terms.

Reply:
I'm using "Lamp" for the triggers you can walk against to activate them and "torch" for the ones you have to ignite with Flame.